components/calendar/src/gregorian.rs

```diff
@@ -163,7 +163,7 @@ impl Date<Gregorian> {
     ///
     /// Years are specified as ISO years.
     ///
-    /// ```rust
+    /// ```rust 
```

Comment:
Run `cargo fmt` to make the CI job pass. It likes to remove trailing whitespace at the end of lines.